tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -224,10 +224,14 @@ Status MetaOptimizer::InitializeOptimizers(
     optimizers->push_back(
         MakeUnique<AutoParallel>(cfg_.auto_parallel().num_replicas()));
   }
+
+#ifndef INTEL_MKL
```

Comment:
This will break the associated unit tests in scoped_allocator_test.cc. Can you please disable those as well for INTEL_MKL?

Reply:
Ok, I turned off the scoped _allocator_optimizer_test when INTEL MKL is enabled. Thanks.